packages/suite/src/actions/suite/metadataActions.ts

```diff
@@ -247,6 +249,43 @@ export const initProvider = () => (dispatch: Dispatch) => {
     return decision.promise;
 };
 
+const getMetadataFilesList = () => async (dispatch: Dispatch) => {
```

Comment:
why is this standalone action? wouldn't it be enough just to call 
` const listResponse = await provider.getMetadataFilesList()` (and do some trimming)

from init action?

Reply:
IMO is neater as a separate action, so that `init` is less bloated 🤷‍♂️